redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/ViewFlexContainer.kt

```diff
@@ -54,6 +56,11 @@ internal class ViewFlexContainer(
   override var modifier: Modifier = Modifier
 
   init {
+    yogaLayout.rootNode.direction = when (hostView.resources.configuration.layoutDirection) {
```

Comment:
```suggestion
    yogaLayout.rootNode.direction = when (hostView.layoutDirection) {
```
https://developer.android.com/reference/android/view/View#getLayoutDirection()

Reply:
If I wanna use that I would need to wait until [the layout direction is resolved](https://developer.android.com/reference/android/view/View#isLayoutDirectionResolved()).